### ISSUE TITLE
VS Code Variables not resolved

### ISSUE THREAD

**Issue by danielniccoli** [Azure/Azurite#2086]
VS Code variables from .vscode\settings.json are not resolved.

![image](https://github.com/Azure/Azurite/assets/2971735/225988ce-b45a-447d-b23a-7250232619dd)

**Comment by blueww** [comment]
@danielniccoli 

Is this issue related with Multiroot workspace in VS Code, as https://github.com/Azure/Azurite/issues/1926.
If so, please track it with that issue.

Else, with only the picture we might don't have enough information to investigate.
(Please understand, we are not Azure function expert.)
So would you please give the detail repro steps, the expected result, and the current result? Then we can look into it.

**Comment by danielniccoli** [comment]
@blueww  No, this issue regards the [Azurite VS Code](https://marketplace.visualstudio.com/items?itemName=azurite.azurite) extension. The explanation with the screenshot should suffice. It makes the problem extremely obvious and reproducible.

**Comment by blueww** [comment]
@danielniccoli 

Thanks for the clarify!

I get you point is: you expect "${workspaceFolder}" be parsed to the real workspace folder, but currently it's taken as original string "${workspaceFolder}".

Azurite get the location with code : https://github.com/Azure/Azurite/blob/525f697c813da95c1327053cc6fac27032a436ce/src/common/VSCEnvironment.ts#L35

We use the VS code API  workspaceConfiguration.get() to get the location. However, the API doesn't convert "${workspaceFolder}" to the real workspace folder. So currently Azurite doesn't support this kind variable in location. 


Azurite welcome contribution!
It would be great if you would raise a PR to support this!

**Comment by danielniccoli** [comment]
Related https://github.com/microsoft/vscode/issues/46471
